src/Field/WfsSearch/WfsSearch.jsx

```diff
@@ -0,0 +1,389 @@
+     */
+    searchAttributes: PropTypes.arrayOf(PropTypes.string).isRequired,
+    /**
+     * The namespace URI used for features.
+     * @type {String}
+     */
+    featureNS: PropTypes.string,
+    /**
+     * The prefix for the feature namespace.
+     * @type {String}
+     */
+    featurePrefix: PropTypes.string,
+    /**
+     * The feature type names. Required.
+     * @type {String[]}
+     */
+    featureTypes: PropTypes.arrayOf(PropTypes.string).isRequired,
+    /**
+     * SRS name. No srsName attribute will be set on geometries when this is not
+     * provided.
+     * @type {string}
+     */
+    srsName: PropTypes.string,
+    /**
+     * The output format of the response.
+     * @type {string}
+     */
+    outputFormat: PropTypes.string,
+    /**
+     * Maximum number of features to fetch.
+     * @type {number}
+     */
+    maxFeatures: PropTypes.number,
+    /**
+     * Geometry name to use in a BBOX filter.
+     * @type {string}
+     */
+    geometryName: PropTypes.string,
+    /**
+     * Optional list of property names to serialize.
+     * @type {String[]}
+     */
+    propertyNames: PropTypes.arrayOf(PropTypes.string),
+    /**
+     * Filter condition. See http://openlayers.org/en/latest/apidoc/ol.format.filter.html
+     * for more information.
+     * @type {object}
+     */
+    filter: PropTypes.object,
+    /**
+     * The ol.map where the map will zoom to.
+     *
+     * @type {Object}
+     */
+    map: PropTypes.instanceOf(OlMap).isRequired,
+    /**
+     * The minimal amount of characters entered in the input to start a search.
+     * @type {Number}
+     */
+    minChars: PropTypes.number,
+    /**
+     * A render function which gets called with the selected item as it is
+     * returned by the server. It must return an `AutoComplete.Option` with
+     * `key={feature.id}`.
+     * The default will display the property `name` if existing or the `id` to.
+     *
+     * @type {function}
+     */
+    renderOption: PropTypes.func,
+    /**
+     * An onSelect function which gets called with the selected feature as it is
+     * returned by server.
+     * The default function will create a searchlayer, adds the feature and will
+     * zoom to its extend.
+     * @type {function}
+     */
+    onSelect: PropTypes.func,
+    /**
+     * Options which are added to the fetch-POST-request. credentials is set to
+     * 'same-origin' as default but can be overwritten.
+     * @type {object}
+     */
+    additionalFetchOptions: PropTypes.object
+  }
+
+  static defaultProps = {
+    srsName: 'EPSG:3857',
+    outputFormat: 'application/json',
+    minChars: 3,
+    additionalFetchOptions: {},
+    /**
+     * Create an AutoComplete.Option from the given data.
+     *
+     * @param {Object} feature The feature as returned by the server.
+     * @return {AutoComplete.Option} The AutoComplete.Option that will be
+     * rendered for each feature.
+     */
+    renderOption: feature => {
```

Comment:
If a user can provide this, we should list actual requirements, if any. I am thinking of the id property, e.g.

Reply:
I'll enhance the docs at the property.